### DISCUSSION TITLE
How i can have Pull Shark

### DISCUSSION THREAD

**Discussion by Houztw** [community/community#19294] [Other Feature Feedback, Questions, & Ideas] [answered]
Help me how to get it please tell me

**Comment by IvailoPetrov06** [comment, marked as answer]
You have to do branch and click pull request and merge it.

**Comment by codemajesty999** [comment]
You have to do branch and click pull request and merge it.

**Comment by ACoastGame** [comment]
Try visiting [this link](https://githubachievements.com/). They have uploaded good content to unlock almost all  the achievements.

**Comment by TGouriSankar** [comment]
You have to do branch and click pull request and merge it.

**Comment by thanhdat2508** [comment]
Visit this website [LINK](https://githubachievements.com/#pull-shark-section), it teach you to reach every badge you want

**Comment by etudiefischer** [comment]
check out a yt video

**Comment by A-S-W-I-N-S-P-A-R-K-Y** [comment]
You have to do branch and click pull request and merge it
